browser/ui/tabs/brave_tab_menu_model.cc

```diff
@@ -152,6 +170,24 @@ void BraveTabMenuModel::Build(Browser* browser,
   if (base::FeatureList::IsEnabled(tabs::features::kBraveRenamingTabs)) {
     BuildItemForCustomization(tab_strip_model, selected_index);
   }
+
+  // Replace SplitTabMenuModel with BraveSplitTabMenuModel.
+  if (arrange_split_view_submenu_) {
```

Comment:
I missed this because I was only looking at codeowner changes, but we should not copy upstream code like this. It has been discussed many times that copying code is much worse than patching. Please do not copy non-trivial code